src/coprocessor/dag/batch/executors/util/scan_executor.rs

```diff
@@ -187,13 +187,18 @@ pub fn field_type_from_column_info(ci: &ColumnInfo) -> FieldType {
 }
 
 /// Checks whether the given columns info are supported.
+/// Enum and Set FieldType should also be valid as we don't need to decode these columns
 pub fn check_columns_info_supported(columns_info: &[ColumnInfo]) -> Result<()> {
```

Comment:
This function now is called only in table / index scan.

Reply:
Please also format your code and run clippy checks by running `make dev`.